lib/bindConnectors/formatInputBody/index.js

```diff
@@ -45,6 +45,11 @@ function formatInputValue(value, schemaProperty) {
 
   schemaProperty = schemaProperty || {};
 
+  // Ignore value for oneOfs
```

Comment:
Point 2 is already being handled, based on your previous feedback. Point 3 is what we're now specifically excluding with this bit of code